### ISSUE TITLE
Support SAM Globals when validating Functions

### ISSUE THREAD

**Issue by sbol-coolblue** [awslabs/cfn-python-lint#157]
I am getting E2531 (You must specify a valid value for runtime) in a SAM template. My Functions do have a Runtime specified, however, this is done via the [Globals](https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst) section. It seems that the parser currently does not yet take this section into account when validating this value (and possibly others?).

```yaml
AWSTemplateFormatVersion: '2010-09-09'
Transform: AWS::Serverless-2016-10-31

Globals:
  Function:
    Runtime: nodejs8.10

Resources:
  ## Snipped Gateway definition ##
  MyExampleFunction:
    Type: AWS::Serverless::Function
    Properties:
      Handler: example.handler
      FunctionName: example-function
      ## No Runtime defined here, is inherited from the Global:Function:Runtime section"
```

Output: `E2531 You must specify a valid value for runtime at nodejs8.10
template.yaml:<row>:<col>`

**Comment by fatbasstard** [comment]
The error message states that the specified value (`nodejs8.10`) is unsupported. Which it was until version `0.2.2` (See the changelog: https://github.com/awslabs/cfn-python-lint/blob/master/CHANGELOG.md#022).

Also running this sample code against the latest codebase gives no errors. Which version of cfn-lint are you running? 

The error message is a bit misformed, created a PR to correct that

**Comment by sbol-coolblue** [comment]
Hi Frank, running cfn-lint version cfn-lint 0.3.1

**Comment by sbol-coolblue** [comment]
You're right, I'm sorry for assuming :) Globals is indeed supported. I've found the real cause of the issue - there's two files called "FunctionRuntime", one living in `/rules/resources/lambda/FunctionRuntime.py` and the other in `/rules/resources/lmbd/FunctionRuntime.py`. That second one did not have nodejs8.10 added to its list. Creating a PR to fix it.

**Comment by fatbasstard** [comment]
Was just debugging with unzipping the actual wheel file (https://pypi.org/project/cfn-lint/0.3.1/#files)

It's a deployment issue, apparantly and "old" folder is still in the artifact. There is no `Lambda` folder in the codebase anymore: https://github.com/awslabs/cfn-python-lint/tree/master/src/cfnlint/rules/resources

So after a rename the "old" one is still there. I'm checking with @kddejong to fix this for a next release

**Comment by sbol-coolblue** [comment]
Jeroen and I reached the exact same conclusion just now :D Thanks for looking into it, Frank!
